src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/message/TextComponent.kt

```diff
@@ -269,13 +270,17 @@ class TextComponent {
                 val linkComponent = MCBaseTextComponent(link).apply {
                     chatStyle = style.createDeepCopy()
 
-                    chatStyle.chatClickEvent = ClickEvent(
-                        ClickEvent.Action.OPEN_URL, if (URI(link).scheme == null) {
-                            "http://$link"
-                        } else {
-                            link
-                        }
-                    )
+                    try {
+                        chatStyle.chatClickEvent = ClickEvent(
+                            ClickEvent.Action.OPEN_URL, if (URI(link).scheme == null) {
+                                "http://$link"
+                            } else {
+                                link
+                            }
+                        )
+                    } catch (ignored: URISyntaxException) {
+                        // this will throw if there is bad uri syntax (e.g. a : with no port)
```

Comment:
I don't really see the point since it most likely wasn't supposed to be a link in the first place.  Just seems like it might spam the console for not really important "errors."

Also, the module creator should notice it didn't work since it isn't clickable if it fails.

Reply:
Fair enough